discovery-provider/alembic/env.py

```diff
@@ -72,6 +77,34 @@ def run_migrations_online():
             context.run_migrations()
 
 
+@compiles(CreateIndex)
+@compiles(CreateTable)
+def _add_if_not_exists(element, compiler, **kw):
```

Comment:
This is sweet! Let's add a link to the github issue used as inspo for this as a comment, for future maintainers